python/tvm/tir/sparse.py

```diff
@@ -28,7 +28,20 @@
 
 class Axis(Object):
     """Base class of all the sparse axes."""
+    @property
+    def name(self):
+        return _ffi_api.GetAxisName(self)
+    
+    @property
+    def length(self):
+        return _ffi_api.GetAxisLength(self)
+    
+    @property
+    def idtype(self):
+        return _ffi_api.GetAxisIndexType(self)
 
+    def __repr__(self):
+        return "name={}, length={}, idtype={}".format(self.name, self.length, self.idtype)
```

Comment:
Repr on Python side is not needed. Let's add ReprPrinter on C++ side later :wink:.